src/libraries/System.Private.CoreLib/src/System/SpanHelpers.T.cs

```diff
@@ -2494,9 +2508,11 @@ internal static int IndexOfAnyValueType<T>(ref T searchSpace, T value0, T value1
         internal static int LastIndexOfAnyValueType<T>(ref T searchSpace, T value0, T value1, T value2, T value3, int length) where T : struct, INumber<T>
```

Comment:
Shouldn't this one be also behind `#if !MONO`?

Reply:
I think you're right, @radekdoulik. This set of overloads wasn't resurrected into `.Mono.cs`, but it's the only of these IndexOf methods still using `where T : struct, INumber<T>` on mono.